### ISSUE TITLE
New submissions for Thursday, 12 September 2024 (showing 29 of 29 entries )

### ISSUE THREAD

**Issue by terrryahn78** [terrryahn78/daily_arxiv_noti#24]
## Development of TiN/AlN-based superconducting qubit components
- **Keywords:** ['superconducting resonators', 'superconducting resonators']
 - **Authors:** Benedikt Schoof, Moritz Singer, Simon Lang, Harsh Gupta, Daniela Zahn, Johannes Weber, Marc Tornow
 - **Subjects:** Subjects:
Applied Physics (physics.app-ph); Materials Science (cond-mat.mtrl-sci); Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2409.07227
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2409.07227
 - **Abstract**
 This paper presents the fabrication and characterization of superconducting qubit components from titanium nitride (TiN) and aluminum nitride (AlN) layers to create Josephson junctions and superconducting resonators in an all-nitride architecture. Our methodology comprises a complete process flow for the fabrication of TiN/AlN/TiN junctions, characterized by scanning electron microscopy (SEM), atomic force microscopy (AFM), ellipsometry and DC electrical measurements. We evaluated the sputtering rates of AlN under varied conditions, the critical temperatures of TiN thin films for different sputtering environments, and the internal quality factors of TiN resonators in the few-GHz regime, fabricated from these films. Overall, this offered insights into the material properties critical to qubit performance. Measurements of the dependence of the critical current of the TiN / AlN / TiN junctions yielded values ranging from 150 ${\mu}$A to 2 ${\mu}$A, for AlN barrier thicknesses up to ca. 5 nm, respectively. Our findings demonstrate advances in the fabrication of nitride-based superconducting qubit components, which may find applications in quantum computing technologies based on novel materials.
## Quantum-Train with Tensor Network Mapping Model and Distributed Circuit Ansatz
- **Keywords:** ['hybrid quantum']
 - **Authors:** Chen-Yu Liu, Chu-Hsuan Abraham Lin, Kuan-Cheng Chen
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2409.06992
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2409.06992
 - **Abstract**
 In the Quantum-Train (QT) framework, mapping quantum state measurements to classical neural network weights is a critical challenge that affects the scalability and efficiency of hybrid quantum-classical models. The traditional QT framework employs a multi-layer perceptron (MLP) for this task, but it struggles with scalability and interpretability. To address these issues, we propose replacing the MLP with a tensor network-based model and introducing a distributed circuit ansatz designed for large-scale quantum machine learning with multiple small quantum processing unit nodes. This approach enhances scalability, efficiently represents high-dimensional data, and maintains a compact model structure. Our enhanced QT framework retains the benefits of reduced parameter count and independence from quantum resources during inference. Experimental results on benchmark datasets demonstrate that the tensor network-based QT framework achieves competitive performance with improved efficiency and generalization, offering a practical solution for scalable hybrid quantum-classical machine learning.
## Scaling of contraction costs for entanglement renormalization algorithms including tensor Trotterization and variational Monte Carlo
- **Keywords:** ['hybrid quantum']
 - **Authors:** Thomas Barthel, Qiang Miao
 - **Subjects:** Subjects:
Strongly Correlated Electrons (cond-mat.str-el); Computational Physics (physics.comp-ph); Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2407.21006
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2407.21006
 - **Abstract**
 The multi-scale entanglement renormalization ansatz (MERA) is a hierarchical class of tensor network states motivated by the real-space renormalization group. It is used to simulate strongly correlated quantum many-body systems. For prominent MERA structures in one and two spatial dimensions, we determine the optimal scaling of contraction costs as well as corresponding contraction sequences and algorithmic phase diagrams. This is motivated by recent efforts to employ MERA in hybrid quantum-classical algorithms, where the MERA tensors are Trotterized, i.e., chosen as circuits of quantum gates, and observables as well as energy gradients are evaluated by sampling causal-cone states. We investigate whether tensor Trotterization and/or variational Monte Carlo (VMC) sampling can lead to quantum-inspired classical MERA algorithms that perform better than the traditional optimization of full MERA based on the exact evaluation of energy gradients. Algorithmic phase diagrams indicate the best MERA method depending on the scaling of the energy accuracy and the number of Trotter steps with the bond dimension. The results suggest substantial gains due to VMC for two-dimensional systems.
